io.github.baldeuniversel.am-okay.metainfo.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+
+  <id>io.github.baldeuniversel.am-okay</id>
+  
+  <name>am-okay</name>
+  <summary>The `am-okay` program allows dynamically to copy/move files/directories (via the terminal)</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-or-later</project_license>
+  
+  <description>
+    <p>
+      The <code>am-okay</code> program allows to copy or move files or directories in a dynamic mode via a console (terminal).
+    </p>
+    <p>
+      With the <code>am-okay</code> program, you can dynamically copy or move files or directories where access is permitted. The program itself partly imitates the Linux <code>cp</code> , <code>mv</code> commands but dynamically. You can tag files or  directories as items to copy or move via a console (terminal), and paste these items via another  console (terminal).
+    </p>
+    <p>
+      A lot of actions are possible, you can even tag files or directories in arrays and paste them  wherever you want in your Linux system.
+    </p>
+  </description>
+  
+  <icon type="stock">io.github.baldeuniversel.am-okay</icon>
+  
+  <categories>
+    <category>Utility</category>
+    <category>FileTools</category>
+  </categories>
+  
+  <provides>
+    <binary>am-okay</binary>
+  </provides>
+
+  <developer id="io.github.baldeuniversel">
+    <name>Baldé Amadou</name>
+    <name xml:lang="en">Baldé Amadou</name>
+  </developer>
+
+  <releases>
+    <release version="3.1.2" date="2024-03-06" />
+  </releases>
+
+  <keywords>
+    <keyword translate="no">utility</keyword>
+    <keyword translate="no">am-okay</keyword>
+    <keyword translate="no">ubuntu</keyword>
+    <keyword translate="no">linux</keyword>
+    <keyword translate="no">terminal</keyword>
+  </keywords>
+
+  <url type="homepage">https://github.com/baldeuniversel/am-okay</url>  
+  <url type="vcs-browser">https://github.com/baldeuniversel/am-okay.git</url> 
+
+  <screenshots>
+    <screenshot type="default">
+      <image>https://drive.google.com/file/d/1y1_Hj0hbxEMNKua7lrvs4aazalv255eU/view?usp=drive_link</image>
```

Comment:
this is not what the review said to do. Using Google drive for that is a bad idea.

also this still doesn't lead to an image either.

Reply:
I tested it in a different web browser, the images were displayed well .

How to do it differently ?

Reply:
Use these links instead:

- `https://raw.githubusercontent.com/baldeuniversel/am-okay/da322a07919a4b74181c7fcfc349926056e9724f/am-okay/usr/local/share/screenshot/screenshot_scenario_1_am-okay.png`
- `https://raw.githubusercontent.com/baldeuniversel/am-okay/da322a07919a4b74181c7fcfc349926056e9724f/am-okay/usr/local/share/screenshot/screenshot_scenario_2_am-okay.png`

Reply:
Done !